pkg/core/transaction/transaction.go

```diff
@@ -38,6 +38,12 @@ type Transaction struct {
 	// Address signed the transaction.
 	Sender util.Uint160
 
+	// Fee to be burned.
+	SystemFee util.Fixed8
```

Comment:
I'm not sure it's a good idea storing them as `util.Fixed8`, but it's OK for now.